repose-aggregator/components/filters/ip-classification/src/main/scala/edazdarevic/commons/net/CIDRUtils.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
yeah, it needs to move to java sources. let me change the question. why was it copy pastad instead of a dependency brought in?